packages/Tag/src/Tag.js

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Close from "@paprika/icon/lib/Times";
+import * as sc from "./Tag.styles";
+import * as types from "./types";
+
+export function Delete({ isDisabled, size, onRemove = () => {} }) {
+  return (
+    <sc.Delete isDisabled={isDisabled} size={size} data-pka-anchor="tag.delete" onClick={onRemove}>
```

Comment:
nit: I suggest `tag.remove` for the anchor name.  This matches better with the prop name `onRemove`.